tests/client/amo/components/TestAddonDetail.js

```diff
@@ -6,8 +6,9 @@ import {
 } from 'react-addons-test-utils';
 import { Provider } from 'react-redux';
 
-import AddonDetail, { allowedDescriptionTags }
-  from 'amo/components/AddonDetail';
+import AddonDetail, {
+  allowedDescriptionTags,
+} from 'amo/components/AddonDetail';
```

Comment:
Ah, indeed, previously it had two imports and needed the spacing? Or something. I'll fix it 👍